### PR TITLE
Animalandpersonforms

### PR DESCRIPTION
**Pull request by Eliran-dev**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+model Animal {
+  id                 String   @id @default(uuid())
+  name               String
+  animalType String @default("אחר")
+  contactName        String
+  contactPhone       String
+  status             String   @default("pending")
+  lastSeen           String?
+  identifyingDetails String?
+  imageUrl           String?
```
Comment (vithano) [inline prisma/schema.prisma:19]: image

File: prisma/schema.prisma [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model Animal {
+  id                 String   @id @default(uuid())
+  name               String
+  animalType String @default("אחר")
```
Comment (vithano) [inline prisma/schema.prisma:13]: type

File: prisma/schema.prisma [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  id                 String  @id @default(uuid())
+  firstName          String
+  lastName           String
+  image              String?
+  contactName        String
+  contactPhone       String
+  status             String  @default("pending") //"unknown" | "found" | "deceased" | "kidnapped";
+  lastSeen           String
+  identifyingDetails String?
+  imageUrl           String?
```
Comment (vithano) [inline prisma/schema.prisma:35]: image and imageUrl?

**Comment by kaminskypavel** [comment]
why did you close the PR?

**Pull request closed without merging**